From akpm@linux-foundation.org Mon Feb  5 16:42:09 2007
From: Adrian Bunk <bunk@stusta.de>
Date: Mon, 05 Feb 2007 16:40:57 -0800
Subject: USB: USB_RTL8150 must select MII
To: greg@kroah.com
Cc: akpm@linux-foundation.org, bunk@stusta.de, rdunlap@xenotime.net
Message-ID: <200702060042.l160g0YW005236@shell0.pdx.osdl.net>


From: Adrian Bunk <bunk@stusta.de>

USB_RTL8150 must select MII to avoid link errors.

Stolen from a patch by Randy Dunlap.

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Cc: "Randy.Dunlap" <rdunlap@xenotime.net>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/usb/net/Kconfig |    1 +
 1 file changed, 1 insertion(+)

--- gregkh-2.6.orig/drivers/usb/net/Kconfig
+++ gregkh-2.6/drivers/usb/net/Kconfig
@@ -84,6 +84,7 @@ config USB_PEGASUS
 config USB_RTL8150
 	tristate "USB RTL8150 based ethernet device support (EXPERIMENTAL)"
 	depends on EXPERIMENTAL
+	select MII
 	help
 	  Say Y here if you have RTL8150 based usb-ethernet adapter.
 	  Send me <petkan@users.sourceforge.net> any comments you may have.
